packages/nuxt-plugin/build.config.ts

```diff
@@ -0,0 +1,46 @@
+import { defineBuildConfig } from "unbuild";
+import { codecovRollupPlugin } from "codecovProdRollupPlugin";
+import packageJson from "./package.json";
+
+export default defineBuildConfig({
+  entries: ["./src/index"],
+  outDir: "dist",
+  declaration: "compatible",
+  sourcemap: true,
+  externals: ["vite"],
+  rollup: {
+    dts: {
+      compilerOptions: {
+        removeComments: false,
+      },
+    },
+    emitCJS: true,
+    esbuild: {
+      minify: true,
+    },
+    replace: {
+      preventAssignment: true,
+      values: {
+        __PACKAGE_VERSION__: JSON.stringify(packageJson.version),
+        __PACKAGE_NAME__: JSON.stringify(packageJson.name),
+      },
+    },
+  },
+  hooks: {
+    "rollup:options": (_ctx, opts) => {
+      if (process.env.PLUGIN_CODECOV_TOKEN && Array.isArray(opts.plugins)) {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+        opts.plugins = [
+          ...opts.plugins,
+          codecovRollupPlugin({
```

Comment:
should this use the nuxt plugin? and set a `bundleName` of `@codecov/nuxt-plugin`?

Reply:
Gonna update to use the name from the package json across the packages, and no this is for checking the size of the plugin itself.